content/blog/build-embedded-swift-application-for-esp32c6/index.md

```diff
@@ -46,19 +46,23 @@ First, let's see the whole build process in Asciinema recording:
    Let's clone the project and configure `TOOLCHAINS` environment variable with the version of the installed Swift 6.
 
 
-   - Linux:
-   ```bash
-   git clone git@github.com:apple/swift-embedded-examples.git --single-branch --branch main
-   cd swift-embedded-examples/esp32-led-strip-sdk
-   ```
+  {{< tabs groupId="config" >}}
+    {{% tab name="macOS" %}}
```

Comment:
In my setting, more software engineers use Linux as opposed to macOS. That's why I would suggest that you promote Linux to the first tab.